docs/src/documentation/03-components/07-interaction/tag/03-accessibility.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: Accessibility
+redirect_from:
+  - /components/tag/accessibility/
+---
+
+## Accessibility
+
+The Tag component has been designed with accessibility in mind, providing proper semantic structure and keyboard navigation for interactive labels that can be selected, clicked, or removed.
+
+### Accessibility Props
+
+**Tag props:**
+
+| Name         | Type   | Description                                                                                                                                                        |
+| :----------- | :----- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| labelDismiss | string | Specifies the accessible name for the dismiss button when `onRemove` is provided. Required for screen reader users to understand the purpose of the remove action. |
+
+### Automatic Accessibility Features
+
+- The component automatically manages ARIA attributes:
+  - When `onClick` is provided, the tag receives `role="button"` and `tabIndex="0"` to make it keyboard accessible
+  - When `onRemove` is provided, the dismiss button receives `role="button"`, `tabIndex="0"`, and `aria-label` from the `labelDismiss` prop
+  - The dismiss button is properly isolated from the main tag interaction to prevent nested interactive elements
+- Focus management is handled automatically:
+  - Interactive tags are included in the tab order with proper focus indicators
+  - The dismiss button can be focused independently from the main tag when both are present
+- Keyboard navigation is fully supported:
+  - **Space** and **Enter** keys activate both the main tag and dismiss button
+  - Event propagation is properly managed to prevent conflicts between tag and dismiss actions
+
+### Best Practices
+
+- Always provide a `labelDismiss` prop when using `onRemove` to ensure screen reader users understand the purpose of the dismiss button.
+- Use clear, descriptive text content for tags to help users understand their meaning and purpose.
+- When using tags for filtering or selection, ensure the selected state is communicated clearly through the visual design and component state.
```

Comment:
Can we give an example how? Do we support this? 🤔

Reply:
Yeah, that looks better.

`Selected` is a weird naming, because it got me immediately thinking about `aria-selected`, which can't be applied to something that is not of role `option` (and a few others, not considered here). But yeah, that's already off-topic here

Reply:
I was playing with it actually, mainly with `aria-pressed`, that we could apply to the _left button_, but I didn't want to change logic stuff. Do you think it makes sense to add it?

Reply:
I am not sure. Is `selected` associated with clickable Tags? If yes, then we can add ariaPressed prop (stateless… devs should handle it). But can one Tag have distinct `selected` states controlled by something else rather than itself (i.e. onClick = undefined)?

Reply:
Anyway, it's off-topic to this docs. If we come to a decision, it should be on a separate task. For that reason, this is approved

Reply:
https://kiwicom.atlassian.net/browse/FEPLT-2751